src/path_functions.cpp

```diff
@@ -33,123 +36,116 @@ QString AOApplication::get_base_path()
 #else
   base_path = applicationDirPath() + "/base/";
 #endif
-
   return base_path;
 }
 
-QString AOApplication::get_data_path()
+QStringList AOApplication::get_path_list()
 {
-  return get_base_path() + "data/";
+  QStringList paths;
+  QString assets_env = qgetenv(ASSET_ENV_VAR);
+  if (assets_env != NULL) {
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
+    QRegExp rx(";");
+#else
+    QRegExp rx(":");
+#endif
```

Comment:
This sounds like feature creep, but can we also have a list of asset paths to be definable via `config.ini`?
Example:
```ini
[AssetFolders]
count = 3
0 = ./base
1 = ./aov
2 = ./casecafe
```
Or simpler:
```ini
asset_folders = base:aov:casecafe
```
The environment variable will obviously take precedence, though.

Reply:
I made precedence effectively go `$AO2_ASSET_PATH:$(grep asset_folders config.ini | cut -f 2 -d =)`.
It searches paths left to right, so anything in the environment variable gets selected first.